BrainPortal/app/models/data_provider.rb

```diff
@@ -859,6 +859,8 @@ def provider_copy_to_otherprovider(userfile, otherprovider, options = {})
     newfile.created_at       = Time.now      unless target_exists
     newfile.updated_at       = Time.now
     newfile.immutable        = false
+    # Check if DP has_browse_path_capabilities? if not, set browse_path to nil
+    newfile.browse_path     = nil if !DataProvider.find(otherprovider.id).has_browse_path_capabilities?
```

Comment:
Why do `DataProvider.find(otherprovider.id)` ?!? Isn't that ridiculous?

Reply:
Indeed it is.